submissions/modupeAdeOnojobi/html-forms/contact.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact</title>
+</head>
+<body>
+    <h1>Sneak Nation</h1>
+    <nav>
+        <ul>
+            <li><a href="../html/index.html">Home</a></li>
+            <li><a href="../html/index.html">Product</a></li>
+            <li><a href="#">About us</a></li>
+            <li><a href="#">Contact Form</a></li>
```

Comment:
You ought to place the link to the contact form here. Both pages should have links to both the contact form and the product page.